lib/core/app/utils/currency_helper.dart

```diff
@@ -28,6 +28,10 @@ abstract class CurrencyHelper {
     return (balance / (pow(10, decimals) * 1.0));
   }
 
+  static double convertToWhole2(double balance, int decimals) {
```

Comment:
How this function differ from the above function ?

Reply:
it takes double instead of int balance